src/test/java/net/raisetechPractice/UserManagementApp/service/UserServiceImplTest.java

```diff
@@ -0,0 +1,106 @@
+package net.raisetechPractice.UserManagementApp.service;
+
+import net.raisetechPractice.UserManagementApp.entity.User;
+import net.raisetechPractice.UserManagementApp.form.UserForm;
+import net.raisetechPractice.UserManagementApp.repository.UserMapper;
+import org.apache.ibatis.javassist.NotFoundException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.Optional;
+
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class UserServiceImplTest {
+    @InjectMocks
+    UserServiceImpl userServiceImpl;
+
+    @Mock
+    UserMapper userMapper;
+
+    @Test
+    public void Userが全件取得できること() throws Exception {
+        when(userMapper.findUsers(null)).thenReturn(List.of(new User(1, "Taro", 30), new User(2, "Jiro", 25)));
+        List<User> actual = userServiceImpl.findUsers(null);
+        assertThat(actual).hasSize(2);
+    }
+
+    @Test
+    public void Userが登録されていないとき空のリストが返ること() {
+        when(userMapper.findUsers(null)).thenReturn(List.of());
```

Comment:
`Collections.emptyList()`を使うのが適切ですね。
```suggestion
        when(userMapper.findUsers(null)).thenReturn(Collections.emptyList());
```

Reply:
`Collections.emptyList()`を使うように修正しました。
https://github.com/soishiki/Lesson13/pull/2/commits/7aa2f2ddbb49b1f292eb730c8b1a7cdb5fb93d64